haruhi_dl/utils.py

```diff
@@ -2225,7 +2212,7 @@ def get_subprocess_encoding():
     return encoding
 
 
-def encodeFilename(s, for_subprocess=False):
```

Comment:
actually, all this function does is assert if the `s` is str and return it, as hdl only runs on py3 now